test/integration/integration_suite_test.go

```diff
@@ -243,6 +243,12 @@ func createClusterManagementAddOn(ctx context.Context) {
 
 	cma := &addonv1alpha1.ClusterManagementAddOn{}
 	err = yaml.Unmarshal(data, cma)
+
+	// Adding installStrategy for testing purposes.
+	cma.Spec.InstallStrategy = addonv1alpha1.InstallStrategy{
+		Type: "Manual",
+	}
```

Comment:
Technically this line should be after we check the error on L252 but practically it doesn't matter.